portfolio/pom.xml

```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
```

Comment:
It doesn't look like you're using Gson yet. Please add the dependency in the PR that uses it. You can either remove the dependency for now or update the Java code to use Gson.